src/transformers/models/roformer/configuration_roformer.py

```diff
@@ -131,3 +135,16 @@ def __init__(
         self.layer_norm_eps = layer_norm_eps
         self.rotary_value = rotary_value
         self.use_cache = use_cache
+
+
+class RoFormerOnnxConfig(OnnxConfig):
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        dynamic_axis = {0: "batch", 1: "sequence"}
```

Comment:
You also need `multiple-choice` dynamic_axis like this: https://github.com/huggingface/transformers/pull/16859/files#diff-b111e7d3999d1220129c285224b7c0100290b6c17534dad32cd4e0b869b17fc0R150

Reply:
Thanks, I didn't look into detail about the supported roformer features, I'll add that now.